ppr-ui/src/components/common/Breadcrumb.vue

```diff
@@ -86,6 +86,16 @@ export default defineComponent({
         const roleBasedBreadcrumbTitle = breadcrumbsTitles[
           getRoleProductCode(getUserRoles.value, getUserProductSubscriptionsCodes.value)
         ]
+        if (localState.isStaff) {
+          const staffBreadcrumbText = 'Staff Dashboard'
+          tombstoneBreadcrumbDashboard[0].text = staffBreadcrumbText
+          tombstoneBreadcrumbDischarge[0].text = staffBreadcrumbText
+          tombstoneBreadcrumbRenewal[0].text = staffBreadcrumbText
+          tombstoneBreadcrumbAmendment[0].text = staffBreadcrumbText
+          tombstoneBreadcrumbRegistration[0].text = staffBreadcrumbText
+          tombstoneBreadcrumbSearch[0].text = staffBreadcrumbText
+          tombstoneBreadcrumbSearchConfirm[0].text = staffBreadcrumbText
+        }
```

Comment:
Not your fault, this is a very verbose computed value. 
But perhaps we refactor the way it returns the `breadcrumbs` value. 

ie create a let property (let breadcrumbs) and assign vs returning it. 
In that case, you would only have to modify the return `breadcrumbs` once at the end.

Sorry for this, we really need to rework the breadcrumb architecture as whole and apply them dynamically.